tutorial.md

```diff
@@ -60,6 +60,92 @@ commit: c91b5bea4830a57eac7882d7455d59518cdf70ec
 spec: 1.0.0-rc2-dev
 ```
 
+### cni
+
+This tutorial will use the latest version of `cni` from the master branch and build it from source.
+
+Download the `cni` source tree:
+
+```
+go get -d github.com/containernetworking/cni
```

Comment:
This has recently changed, and the CNI plugins are now under the plugins directory.
So this should be:
`go get -d github.com/containernetworking/plugins`